homeassistant/components/intellifire/coordinator.py

```diff
@@ -69,7 +54,6 @@ def device_info(self) -> DeviceInfo:
             manufacturer="Hearth and Home",
             model="IFT-WFM",
             name="IntelliFire",
-            identifiers={("IntelliFire", f"{self.read_api.data.serial}]")},
-            sw_version=self.read_api.data.fw_ver_str,
-            configuration_url=f"http://{self._api.fireplace_ip}/poll",
+            identifiers={("IntelliFire", f"{self.fireplace.serial}]")},
```

Comment:
For a future PR, we should migrate the first part of the identifier `IntelliFire` to be migrated to the lowercase domain name `intellifire`

Reply:
I'm leaving this unresolved so I dont forget ... once we get the other stuff resolved ... I'll close this one if thats cool